main.go

```diff
@@ -97,6 +105,19 @@ func runMain() error {
 	}
 }
 
+func shouldTurnOff(minutesUntilActive int) bool {
+	if time.Now().Before(stayOnUntil) {
+		log.Println("delayed because of stayOnUntil")
```

Comment:
instead of logging here every loop around, how about just logging the time it'll be kept on until in SetStayOnUntil? That'll be less noisy while still providing more useful information.